ignite/contrib/metrics/regression/r2_score.py

```diff
@@ -20,6 +20,16 @@ class R2Score(_BaseRegression):
     - ``update`` must receive output of the form ``(y_pred, y)`` or ``{'y_pred': y_pred, 'y': y}``.
     - `y` and `y_pred` must be of same shape `(N, )` or `(N, 1)` and of type `float32`.
 
+    Args:
```

Comment:
`__init__` useless ?

Reply:
we still need to do super().__init__, right?

Reply:
Same than above.